### PR TITLE
[README] Add citation

### PR DESCRIPTION
**Pull request by jgbarah**
Add a citation reference for the paper on GrimoireLab.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,3 +134,27 @@ folder includes instrucctions and configuration files to deploy GrimoireLab usin
...
+
+```
+Dueñas S, Cosentino V, Gonzalez-Barahona JM, del Castillo San Felix A, Izquierdo-Cortazar D, Cañas-Díaz L, Pérez García-Plaza A. 2021. GrimoireLab: A toolset for software development analytics. PeerJ Computer Science 7:e601 https://doi.org/10.7717/peerj-cs.601
+```
+
+BibTeX / BibLaTeX:
+
+```
+@Article{duenas2021:grimoirelab,
+  author = 	 {Dueñas, Santiago and Cosentino, Valerio and Gonzalez-Barahona, Jesus M. and del Castillo San Felix, Alvaro and Izquierdo-Cortazar,  Daniel and Cañas-Díaz Luis and Pérez García-Plaza, Alberto},
```
Comment (sanacl) [inline README.md:152]: I guess there is a missing comma between my first and second names

**Review by sanacl — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jgbarah) [reply README.md]: Of course you're right. Sorry for the mistake. Fixed.

**Comment by sanacl** [comment]
@jgbarah LGTM.

**Comment by jgbarah** [comment]
@sanacl could you please clear your request, if you think it is done? Just to have a clean evaluation, and see if somebody does the merge.

**Comment by sduenas** [comment]
@jgbarah I was waiting for Venu to do this. I think he should be responsible of adding this kind of content as he is the community manager. I will talk to him about this today.

**Review by vchrombie — APPROVED** [review]
LGTM :+1:

**Comment by vchrombie** [comment]
@jgbarah sorry for delaying the review.
Do you mind rebasing the branch so that we can proceed with the merge?

Thanks in advance.

**Comment by vchrombie** [comment]
Thanks, @jgbarah! :smiley:

**Pull request merged**